src/Norms.js

```diff
@@ -12,7 +12,6 @@ export default class Norms extends React.Component {
           <ul>
             <li>Follow NJIT’s code of ethics</li>
             <li>Mute yourself</li>
-            <li>Put pronouns as part of your Zoom names</li>
```

Comment:
Don't delete this line

Reply:
dude. srsly

Reply:
@jvannier when you get a chance i fixed it 🦜